build-system/runner/src/org/ampproject/AmpCodingConvention.java

```diff
@@ -48,20 +47,20 @@ public AmpCodingConvention(CodingConvention convention) {
   @Override
   public ImmutableCollection<AssertionFunctionSpec> getAssertionFunctions() {
     return ImmutableList.of(
-      AssertionFunctionSpec.makeReturnTypeAssertion(
-          "module$src$log.devAssert"),
-      AssertionFunctionSpec.makeReturnTypeAssertion(
-          "devAssert$$module$src$log"),
-      AssertionFunctionSpec.makeReturnTypeAssertion(
-          "module$src$log.userAssert"),
-      AssertionFunctionSpec.makeReturnTypeAssertion(
-          "userAssert$$module$src$log"),
-      AssertionFunctionSpec.makeReturnTypeAssertion(
-          "assertService$$module$src$element_service"),
-      AssertionFunctionSpec.makeReturnTypeAssertion(
-          "module$src$layout.assertLength"),
-      AssertionFunctionSpec.makeReturnTypeAssertion(
-          "assertLength$$module$src$layout")
+      AssertionFunctionSpec.forMatchesReturn().setFunctionName(
```

Comment:
If we're upgrading Closure Compiler, can we switch to https://github.com/google/closure-library/blob/master/closure/goog/asserts/asserts.js#L181?

Reply:
Good idea. Trying this. Shouldn't these be `@closurePrimitive {asserts.matchesReturn}` though?

https://github.com/google/closure-library/blob/master/closure/goog/asserts/asserts.js#L212

Reply:
It appears so https://github.com/google/closure-compiler/blob/004024fce9c307e8855c8eaa376d556fb344e763/src/com/google/javascript/rhino/ClosurePrimitive.java#L57-L59

Reply:
Yep, I see the same set of options in the [v20190513](https://github.com/google/closure-compiler/blob/maven-v20190513/src/com/google/javascript/rhino/ClosurePrimitive.java#L57-L59) codebase. Trying this.

Reply:
That worked! `v0.js` is exactly the same before and after this change. See 5ebcee3cd569d11295cc8f0dda1ee0a17c7b222c.